### ISSUE TITLE
Blog Post Ideas

### ISSUE THREAD

**Issue by amitness** [amitness/amitness.github.io#47]
Rough sketches of possible blog posts.

- Periodic cron tasks with github actions

ML:
- Noisy label learning intro + finding labeling errors
- Sentence Embeddings + intent detection based on low training phrases
- ~Understanding Object Oriented Programming Concepts in Pytorch~

Software engineering:
- making ML projects more reproducible: os.path.join, hardcoded path -> env variable, pip->git,  pickle files (decouple)
- How you're reinventing the wheel in Python
- ~Migration guide from os.path to Pathlib~

Illustrations:
- Factorization Machines explained with a simple use-case
- ~ALBERT: [code samples](https://github.com/dhlee347/pytorchic-bert/compare/master...graykode:master)~
- NLP Standard Tasks Illustrated
- Illustrated Guide to Using RNNs in Keras*
- Sklearn hyperparameters visualized for each estimator
- Survey of semi-supervised learning methods (learning from unlabeled data): https://drive.google.com/file/d/1ax1-XprJHDRRv2Ru3dJwPLs3ShxcpQ3r/view

NLP:
- Markov chains and their application to gibberish detection
Applied BERT series:
    - Filling missing words in a sentence using BERT
    - Leveraging BERT for Semantic Grouping of Vocabulary
    - NLP transfer learning practical use-cases
- Industrial-scale spelling correction with hunspell and spacy
- Custom Tokenizer (BERT/Spacy) in Scikit-learn vectorizers
- Using a Wikipedia corpus to train X
- Finetuning deepmoji for sentiment detection
- BERT for processing/embedding multi-lingual data
- Practical Guide to Dependency Parsing in Spacy
